tests/isosurfaces_simple_box_2D_custom_fieldnames.prm

```diff
@@ -0,0 +1,17 @@
+# This input file tests the isosurface mesh refinment plugin
+include $ASPECT_SOURCE_DIR/tests/isosurfaces_simple_box_2D.prm
+
+subsection Compositional fields
+  set Number of fields = 3
+  set Names of fields = field1, FieldB, fIEld_C
+end
+
+subsection Mesh refinement
+  set Initial global refinement          = 0
+  set Initial adaptive refinement        = 2
+  set Strategy = isosurfaces
+  subsection Isosurfaces
+    set Isosurfaces = 4,    4,    Temperature: 0 | 600,   FieldB : 0.0 | 0.5 ;\
```

Comment:
Maybe you can add a comment here that although the isosurface entry specifies min/max level 4, refinement level will never be more than 2 because initial global + initial adaptive = 2